### PR TITLE
Added BGP support on Windows.

### PR DESCRIPTION
**Pull request by song-jiang**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: getting-started/windows-calico/troubleshoot.md [hunk 1]
```diff
@@ -104,3 +104,65 @@ If the error includes 'loading config file "<path-to-kubeconfig>"', follow the i
...
+expected IP of a node or an external BGP peer. If the IP of the failed peering is a Windows node:
+
+- Check that the node is up a reachable over IP
+- Check that the RemoteAccess service is installed and running:
+
+  ```powershell
+  PS C:\> Get-Service | ? Name -EQ RemoteAccess
+  ``` 
+- Check the logs for the confd service in the configured log directory for errors
+(default C:\TigeraCalico\logs).
```
Comment (lmm) [inline getting-started/windows-calico/troubleshoot.md:126]: I wonder if we are at the point where we can add a variable for the Windows root dir name to _config.yml like we did with `site.prodnameWindows`: https://github.com/projectcalico/calico/blob/fc70a4f61ee13d8e63207bd778c20e8b1bcdc6f6/_config.yml#L21

**Review by lmm — COMMENTED** [review]
Looking good, added some comments

File: getting-started/windows-calico/troubleshoot.md (hunk 1 above)
Comment (song-jiang) [reply getting-started/windows-calico/troubleshoot.md:126]: Added `rootDirWindows: C:\CalicoWindows`

File: getting-started/windows-calico/limitations.md [hunk 2]
```diff
@@ -22,7 +22,67 @@ canonical_url: '/getting-started/windows-calico/limitations'
...
+      valid_lft forever preferred_lft 
+      forever 
+```    
+In this case, the IPv4 is 192.168.171.136/24; which, after applying the /24 mask gives 192.168.171.0/24 for the IP network. 
+
+Similarly, on Windows, you can run 
+
+```
+PS C:\> 
+ipconfig
```
Comment (lmm) [inline getting-started/windows-calico/limitations.md:65]: nit: The `ipconfig` should be on the same line as the prompt

File: getting-started/windows-calico/limitations.md [hunk 3]
```diff
@@ -22,7 +22,67 @@ canonical_url: '/getting-started/windows-calico/limitations'
...
+    link/ether 00:0c:29:cb:c8:19 brd ff:ff:ff:ff:ff:ff 
+    inet 192.168.171.136/24 brd 192.168.171.255 scope 
+
+    global eth0 
+      valid_lft forever preferred_lft forever 
+      inet6 fe80::20c:29ff:fecb:c819/64 scope 
+      link 
+
+      valid_lft forever preferred_lft 
+      forever
```
Comment (lmm) [inline getting-started/windows-calico/limitations.md:57]: nit: the output has a weird indent

**Review by lmm — APPROVED** [review]
Left some minor comments but LGTM

**Pull request merged**